src/components/Button/index.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Button from '.';
+
+const meta = {
+  title: 'Components/Button',
+  component: Button,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'centered',
+  },
+  argTypes: {
+    shape: {
+      options: ['solid', 'outlined'],
+      control: { type: 'select' },
+    },
+    round: {
+      options: ['xl', '3xl'],
+      control: { type: 'select' },
+    },
+    size: {
+      options: ['lg', 'sm', 'xs'],
+      control: { type: 'select' },
+    },
+    disabled: {
+      control: { type: 'boolean' },
+    },
+  },
+} satisfies Meta<typeof Button>;
```

Comment:
```suggestion
const meta: Meta<typeof Button> = {
  title: 'Components/Button',
  component: Button,
  tags: ['autodocs'],
  parameters: {
    layout: 'centered',
  },
  argTypes: {
    shape: {
      options: ['solid', 'outlined'],
      control: { type: 'select' },
    },
    round: {
      options: ['xl', '3xl'],
      control: { type: 'select' },
    },
    size: {
      options: ['lg', 'sm', 'xs'],
      control: { type: 'select' },
    },
    disabled: {
      control: { type: 'boolean' },
    },
  },
};
```
[Suggestion] 이렇게 쓸 수 있을거 같습니다!